general/odoo_basics/add_user.rst

```diff
@@ -1,46 +1,141 @@
-=================
-How to add a user
-=================
+==================================
+Add Users and Manage Access Rights
+==================================
 
-Odoo provides you with the option to add additional users at any given
-point.
+Odoo defines a *user* as someone who has access to a database to perform daily tasks. You can add as
+many users as you need, and in order to restrict the type of information each user can access, rules
+can be applied. Users and access rights can be added and changed at any point.
```

Comment:
Maybe you should explain that new users will change the db subscription, no?
I'm currently working on a doc about users. Maybe you could link this doc to the old version of the doc (I won't change the link): https://www.odoo.com/documentation/user/13.0/db_management/documentation.html

Reply:
I saw that you put it at the end of the doc (see also) but, seeing the importance of that info, I would put it here instead of at the end.

Reply:
- Yes, it is mention in the doc that the number of users changes the subscription price.
- I believe at the end is enough and follows a logical flow (: